### PR TITLE
Fix the default option

### PR DESCRIPTION
**Pull request by shalevr**

### CODE REVIEW

File: cisco_telescope/instrumentations/utils.py [hunk 1]
```diff
@@ -23,9 +23,10 @@ def set_payload(
                     consts.ENCODING_UTF8, consts.DECODE_PAYLOAD_IN_CASE_OF_ERROR
                 )
             else:
-                payload_decoded = payload or ""
+                payload_decoded = payload
```
Comment (haddasbronfman) [inline cisco_telescope/instrumentations/utils.py:26]: a suggestion: I would do like this:

        payload_decoded = "payload"
        payloads_enabled = Configuration().payloads_enabled
        max_payload_size = Configuration().max_payload_size

        if payloads_enabled or (attr not in PAYLOAD_ATTRIBUTES):
            if isinstance(payload, bytes):
                payload_decoded = payload.decode(
                    consts.ENCODING_UTF8, consts.DECODE_PAYLOAD_IN_CASE_OF_ERROR
                )
            span.set_attribute(attr, payload_decoded[:max_payload_size])

File: cisco_telescope/instrumentations/utils.py [hunk 2]
```diff
@@ -39,4 +40,5 @@ def add_flattened_dict(span: Span, prefix, attributes: dict):
             for attribute_key, attribute_value in Utils.lowercase_items(
                 attributes
             ).items():
-                span.set_attribute(f"{prefix}.{attribute_key}", attribute_value)
+                if attribute_value:
```
Comment (haddasbronfman) [inline cisco_telescope/instrumentations/utils.py:43]: What if the user added an empty header intentionally? Why we omit it?

File: cisco_telescope/tracing.py [hunk 3]
```diff
@@ -33,8 +33,8 @@
 def init(
     service_name: str = None,
     cisco_token: str = None,
-    debug: bool = False,
-    payloads_enabled: bool = False,
+    debug: bool = None,
+    payloads_enabled: bool = None,
```
Comment (haddasbronfman) [inline cisco_telescope/tracing.py:37]: I would use the default value from 'Consts', it's more readable to my opinion and save code later.

File: cisco_telescope/instrumentations/utils.py (hunk 2 above)
Comment (shalevr) [reply cisco_telescope/instrumentations/utils.py:43]: Right, Fixed

Comment (shalevr) [reply cisco_telescope/instrumentations/utils.py]: Fixed

File: cisco_telescope/tracing.py (hunk 3 above)
Comment (shalevr) [reply cisco_telescope/tracing.py:37]: we talked about that

**Review by haddasbronfman — APPROVED** [review]

**Pull request merged**